catalog/catalog.yaml

```diff
@@ -377,6 +401,8 @@ components:
           $ref: '#/components/schemas/Support'
         providerOpts:
           $ref: '#/components/schemas/ProviderOpts'
+        credentialsHealthCheck:
```

Comment:
what about `authHealthCheck`? Since we already have a field called `authType`, so it would be good to recycle the word `auth` to be consistent. Alternatively, `connectionHealthCheck` could also work.

Reply:
yeah that works